android/sdk/build.gradle

```diff
@@ -75,6 +75,7 @@ dependencies {
     implementation project(':react-native-webrtc')
     implementation project(':react-native-webview')
     implementation project(':react-native-splash-screen')
+    implementation project(':react-native-slider')
```

Comment:
Can you please sort this alphabetically? Looks like we also missed it with splash-screen.

Reply:
Fixed.